scripts/jenkins/build

```diff
@@ -52,6 +55,31 @@ fi
 
 ./autogen.sh |& tee $ARTIFACTS/configure.log
 
+if [[ $TRANSPORT == "lnet" ]]; then
+	TRANSPORT='lustre-client'
+else
+	echo "No need to change TRANSPORT value"
+fi
+
+# Checking if package is available or not
+if [[ ! -z $(rpm -qa $TRANSPORT) ]]; then
+        echo "$TRANSPORT package is present"
+else
+        echo "need to install $TRANSPORT module"
+	yum install ${TRANSPORT}* -y
+fi
+
+if [[ "$TRANSPORT" == "libfabric" ]]; then
+        echo "You are using default $TRANSPORT module"
+else
+	if  [[ ! -z $(rpm -qa | grep libfabric) ]]; then
+        	rpm -e $(rpm -qa | grep libfabric)
+	fi
+	sed -i '/libfabric/d' cortx-motr.spec.in
```

Comment:
What do you think about using the power of autoconf instead of changing the source code with sed?

<details>
<summary>
Here is an example:
</summary>

```

diff --git a/configure.ac b/configure.ac
index 0a21812f..3e87c8f1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -922,6 +922,7 @@ AS_IF([! test -z "$with_libfab"],
         LIB_FAB_INCLUDES="-I$LIBFABRIC/include"
         LIBFAB_LIBS="-L$LIBFABRIC/lib -lfabric"
         M0_CPPFLAGS="-DENABLE_LIBFAB $M0_CPPFLAGS $LIB_FAB_INCLUDES"
+        BUILD_DEPEND_LIBFAB=""
       ],
 )
 AS_IF([test x$enable_libfab = xyes || test "x$with_default_network" = "xLF" ],
@@ -948,6 +949,7 @@ AS_IF([ test -z "$with_libfab" && test x$enable_libfab = xno],
                        M0_CPPFLAGS="-DENABLE_LIBFAB $M0_CPPFLAGS"
                        enable_libfab=yes
                        AC_DEFINE_UNQUOTED([M0_DEFAULT_NETWORK], ["LF"])
+                      BUILD_DEPEND_LIBFAB="BuildRequires: libfab-devel"
                      ]
                )
              ], []
@@ -955,9 +957,9 @@ AS_IF([ test -z "$with_libfab" && test x$enable_libfab = xno],
       ]
 )
 AC_SUBST([LIBFAB_LIBS])
+AC_SUBST([BUILD_DEPEND_LIBFAB])
 AM_CONDITIONAL([ENABLE_LIBFAB],
                [test "x$enable_libfab" = xyes || ! test -z "$with_libfab"])
-
 #
 # Checking Lustre --------------------------------------------------------- {{{1
 #
diff --git a/cortx-motr.spec.in b/cortx-motr.spec.in
index 41379bc4..4707e4c1 100644
--- a/cortx-motr.spec.in
+++ b/cortx-motr.spec.in
@@ -124,7 +124,7 @@ BuildRequires:  python-devel
 %else
 BuildRequires:  python2-devel
 %endif
-BuildRequires:  libfabric-devel
+@BUILD_DEPEND_LIBFAB@
 BuildRequires:  libedit-devel
 %if %{with isal}
 BuildRequires:  isa-l

```
</details>

Reply:
@upendrapatwardhan : check the above example.

Reply:
Thanks for the suggestion Ivan. Will try to add this.